ophyd/sim.py

```diff
@@ -122,10 +127,17 @@ def sleep_and_finish():
                     st._finished()
 
                 threading.Thread(target=sleep_and_finish, daemon=True).start()
+
             return st
         else:
             self.put(self._func())
-            return NullStatus()
+            return ADTriggerStatus(self, done = True, success = True)
+
+    def stage(self):
+        pass
```

Comment:
I don't think we have settled the question of whether Signals should implement stage/unstage, which is currently limited to `Device`. If it _is_ added, it should be added to all Signals (on `Signal`), not here.

Reply:
It is only done on simulated signals, and was required for testing purposes (the motor axes in the sim devices are based on these signals). Normal signals do not have these attrtibutes.